simple/templates/ingress.yaml

```diff
@@ -6,7 +6,18 @@ metadata:
   annotations:
 {{ toYaml .Values.ingress.annotations | indent 4 }}
 spec:
-  backend:
-    serviceName: {{ .Values.name }}
-    servicePort: {{ .Values.ingress.servicePort }}
+  rules:
+    {{- range .Values.ingress.rules }}
+    - 
+      {{- if .host}}
+      host: {{ .host }}
+      {{- end }}
+      http:
+        paths:
+        {{- range .http.paths }}
+        - backend:
```

Comment:
should also add `path` here, sounds will be commonly used

Reply:
done